src/main/java/thePackmaster/cards/basicspack/StrikePlus.java

```diff
@@ -0,0 +1,43 @@
+package thePackmaster.cards.basicspack;
+
+import com.evacipated.cardcrawl.mod.stslib.cards.interfaces.OnObtainCard;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardAndObtainEffect;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.cards.Defend;
+import thePackmaster.cards.Strike;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class StrikePlus extends AbstractPackmasterCard {
+    public final static String ID = makeID("StrikePlus");
+
+    public StrikePlus() {
```

Comment:
Hmm, having skills that are strikes are kinda jank. Would you mind renaming it? if not, at least give it the strike tag for consistency